### DISCUSSION TITLE
Dagster ECS fargate configuration per ops rather than the full job

### DISCUSSION THREAD

**Discussion by mmarionncern** [dagster-io/dagster#24783] [Q&A] [answered]
Hello Dagster experts, 

I have an industrial use case where a dagster job is split into two set of ops, with one type of ops requiring much larger resources than the second type (typically 32 threads vs 1 thread).
So far I dealt with it using dagster-celery queues running on different dedicated machines, but for cost reduction, I would like to move to an AWS Fargate like resource management as it is available on dagster :
https://docs.dagster.io/deployment/guides/aws

However, resources seem to be defined once for the full job. Is it possible somehow to define the resources for each ops instead?

Thank you by advance

**Comment by garethbrickman** [comment, marked as answer]
I think this would require the ECS task executor in this feature request: https://github.com/dagster-io/dagster/issues/9671

**Reply by mmarionncern** [reply]
got it, thanks!
